Source/WebCore/workers/WorkerMessagingProxy.cpp

```diff
@@ -78,12 +78,13 @@ class WorkerUserGestureForwarder : public ThreadSafeRefCounted<WorkerUserGesture
         m_token = nullptr;
     }
 
-    UserGestureToken* userGestureToForward() const
+    RefPtr<UserGestureToken> userGestureToForward() const
     {
         ASSERT(isMainThread());
-        if (!m_token || m_token->hasExpired(UserGestureToken::maximumIntervalForUserGestureForwarding))
+        RefPtr token = m_token;
+        if (!token || token->hasExpired(UserGestureToken::maximumIntervalForUserGestureForwarding))
```

Comment:
Unfortunate. Static analysis cannot recognize that `hasExpired()` is trivial because `MonotonicTime::now()` is not inline (cc @rniwa)